portfolio/src/main/webapp/script.js

```diff
@@ -13,28 +13,55 @@
 // limitations under the License.
 
 
-/**
- * @return {string} 
- * a random greeting
+/* @return {string}
+ * Generate a random greeting
  */
 function generateRandomGreeting() {
     //TODO use google translated to get greetings in different languages
     const greetings =
         ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
     // Pick a random greeting.
-    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
     return greeting;
 }
 
+/* Retrieve user comments and display them */
+async function getUserComments(){
+    try{
+    const response = await fetch("/data");
+    const data = await response.json();
+
+    const commentEl = document.querySelector("#user-comments");
+        if(typeof(commentEl) != 'undefined' && commentEl != null){
+            for(var comment in data){
```

Comment:
Styleguide prefers using "const" whenever possible, and when it is not a const variable, using "let" instead of "var"
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let